platforms/hyperledger-fabric/configuration/samples/network-fabricv2_5.yaml

```diff
@@ -0,0 +1,625 @@
+##############################################################################################
```

Comment:
Do we need a new format of network.yaml for v2.5 ? If possible, can we not have the descriptions updated in the original network.yaml sample file itself?
(Checking the network-schema.json it seems we can use the same network.yaml. Correct me if I am wrong here)

Reply:
Yes, the original could be used by adding the osn_creator_org.name field in the channel section
<img width="710" alt="image" src="https://github.com/hyperledger/bevel/assets/83813093/f96822b2-19d7-424b-90b2-161711677e96">